<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgKfMenu</class>
 <widget class="QDialog" name="dlgKfMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Поточна стадія</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="le_State"/>
   </item>
   <item>
    <widget class="QPushButton" name="bn_13">
     <property name="text">
      <string>Стоп</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bn_05">
     <property name="text">
      <string>Висолодження</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bn_06">
     <property name="text">
      <string>Промивка</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bn_07">
     <property name="text">
      <string>Продувка</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bn_08">
     <property name="text">
      <string>Сушіння</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="bn_09">
     <property name="text">
      <string>Вивантаження</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="bnExit">
     <property name="text">
      <string>Закрити</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bnExit</sender>
   <signal>clicked()</signal>
   <receiver>dlgKfMenu</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
